### ISSUE TITLE
Cannot read property 'string' of undefined

### ISSUE THREAD

**Issue by RaviDhiman0009** [mashmatrix/react-lightning-design-system#240]
when i'm import the module ( import { Button } from 'react-lightning-design-system'; ) i got the error on my browser window "Cannot read property 'string' of undefined". even i am not using it.
![lightingdesign](https://user-images.githubusercontent.com/33057966/31943796-8433e024-b8e7-11e7-9262-13360e9f3a14.png)

**Comment by bondz** [comment]
Yes, because PropTypes were [deprecated in React 15.5](https://reactjs.org/blog/2017/04/07/react-v15.5.0.html) and removed in React 16.

**Comment by itsrifat** [comment]
Still happening in `v2.4.5`. There are still references to `React.PropTypes` here

https://github.com/mashmatrix/react-lightning-design-system/blob/6e1f587b5492faa233f537937ba08a132bde3d66/src/scripts/Table.js#L167

and here:
https://github.com/mashmatrix/react-lightning-design-system/blob/1e80f8784933ebc7fdc21b9421570e305eb7d23c/stories/GridStories.js#L16

**Comment by bondz** [comment]
Not sure if master has been pushed to NPM yet.

**Comment by rbx44** [comment]
^ https://github.com/mashmatrix/react-lightning-design-system/issues/252
